MAUI/Cartesian-Charts/migration.md

```diff
@@ -788,23 +788,32 @@ chart.ChartBehaviors.Add(selectionBehavior);
 
 <chart:SfCartesianChart>
 . . .
-    <chart:SfCartesianChart.SelectionBehavior>
-        <chart:ChartSelectionBehavior />
-    </chart:SfCartesianChart.SelectionBehavior>
-
-    <chart:ColumnSeries SelectionBrush="Green" 
-            SelectedIndex="2"
-            ItemsSource="{Binding Data}" 
-            XBindingPath="Demand" 
-            YBindingPath="Year2010" />
+    <chart:SfCartesianChart.Series>
```

Comment:
@Sowndharya1911 , use series selection code, and add one line below this content. 


For more information on selection behavior check [here](https://help.syncfusion.com/maui/cartesian-charts/selection)